mail_tracking/views/mail_tracking_email_view.xml

```diff
@@ -0,0 +1,119 @@
+                        <field name="partner_id"/>
+                        <field name="recipient"/>
+                        <field name="sender"/>
+                    </group>
+                    <group>
+                        <field name="timestamp"/>
+                        <field name="time"/>
+                        <field name="date"/>
+                    </group>
+                </group>
+                <group attrs="{'invisible': [('bounce_type', '=', False)]}">
+                    <field name="bounce_type"/>
+                    <field name="bounce_description"/>
+                </group>
+                <group attrs="{'invisible': [('error_type', '=', False)]}">
+                    <field name="error_smtp_server"
+                           attrs="{'invisible': [('error_smtp_server', '=', False)]}"/>
+                    <field name="error_type"/>
+                    <field name="error_description"/>
+                </group>
+                <label for="tracking_event_ids"/>
+                <div>
+                    <field name="tracking_event_ids">
+                        <tree string="Tracking events" colors="grey:event_type in ('deferral');black:event_type in ('send');red:event_type in ('hard_bounce', 'soft_bounce', 'spam', 'reject');blue:event_type in ('unsub', 'click', 'open')">
+                            <field name="time"/>
+                            <field name="event_type"/>
+                            <field name="ip"/>
+                            <field name="url"/>
+                            <field name="user_country_id" string="Country"/>
+                            <field name="os_family" string="OS"/>
+                            <field name="ua_family" string="User agent"/>
+                        </tree>
+                    </field>
+                </div>
+            </sheet>
+        </form>
+    </field>
+</record>
+
+<record model="ir.ui.view" id="view_mail_tracking_email_tree">
+    <field name="name">mail.tracking.email.tree</field>
+    <field name="model">mail.tracking.email</field>
+    <field name="arch" type="xml">
+        <tree string="MailTracking emails" colors="grey:state in (False, 'deferred');black:state in ('sent', 'delivered');green:state in ('opened');red:state in ('rejected', 'spam', 'bounced', 'soft-bounced');blue:state in ('unsub')">
+            <field name="state" invisible="1"/>
+            <field name="time"/>
+            <field name="date" invisible="1"/>
+            <field name="name"/>
+            <field name="sender" string="Sender"/>
+            <field name="recipient" string="Recipient"/>
+        </tree>
+    </field>
+</record>
+
+<record model="ir.ui.view" id="view_mail_tracking_email_search">
+    <field name="name">mail.tracking.email.search</field>
+    <field name="model">mail.tracking.email</field>
+    <field name="arch" type="xml">
+        <search string="MailTracking email search">
+            <field name="sender" string="Email"
```

Comment:
It is included in filter_domain attribute. This direct filter search in sender and recipient fields